observability/cockpit/how-to/create-token.mdx

```diff
@@ -0,0 +1,43 @@
+---
+meta: How to create a token
+  title: How to create a token
+  description: This page shows you how to create a token
+content:
+  h1: How to to create a token
+  paragraph: This page shows you how to create a token
+categories:
+  - observability
+dates:
+  validation: 2022-10-17
+  posted: 2022-10-30
+---
+
+This page shows you how to create [tokens](/observability/cockpit/concepts/#tokens), to grant other cloud providers’ services and platforms access to your [Cockpit](/observability/cockpit/concepts/#cockpit).
+
+<Message type="requirement">
+
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+
+</Message>
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
+
+3. Click the **Tokens** tab.
+
+4. Click **Generate new token** to create a token. A pop-up displays prompting you to create a token.
+
+5. Choose a name for your token or use the already-generated name.
+
+6. Select the permissions you wish to apply to your token.
```

Comment:
Add a message type important to warn the user they must select at least one permission